roles/webots/robocup.role

```diff
@@ -9,6 +9,8 @@ nuclear_role(
   support::configuration::GlobalConfig
   # Network
   network::PlotJuggler
+  network::NUClearNet
+  network::NetworkForwarder
```

Comment:
do we want these in here

Reply:
Probably best for testing behaviour, we can remove later